test/sequential/test-child-process-pass-fd.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const fork = require('child_process').fork;
+const net = require('net');
+
+const N = 80;
```

Comment:
I don't know if it's a good idea to spawn 80 processes.  You might hit the ulimit on some systems and it will be very slow on the less powerful buildbots.  It takes about 500 ms to start a process on the rpi1.